Project finished

jollejonas opened the pull request:
Looking forward to your review, thanks 😄

jollejonas committed:
Project finished

TheCSharpAcademy reviewed (changes requested):
@jollejonas Thanks for submitting! Looks neat! 👌🏻Just one problem: Add and Delete work great but Update isn't working for me. Does it work for you?

jollejonas committed:
UpdateRecord function didn't work, as a '=' sign was missing in the asp-for="Expense.Id".
Because of this when you tried to update an item, it wouldn't have the Id and therefor selected Id 0.

jollejonas commented:
@TheCSharpAcademy 
Thanks for the review. I have updated the project, so now the update function works again.

TheCSharpAcademy commented:
@jollejonas Thanks for fixing it, project approved! 🫡
